cmd/crictl/sandbox.go

```diff
@@ -147,8 +147,8 @@ var podSandboxStatusCommand = cli.Command{
 }
 
 var listPodSandboxCommand = cli.Command{
-	Name:  "sandboxes",
-	Usage: "List sandboxes",
+	Name:  "pods",
```

Comment:
`pods` -> `podsandboxes`  ?

Reply:
This command will be frequently used, I feel like `podsandboxes` is too long, which will be very annoying in the future. :P

Reply:
`pods` looks more simple

Reply:
Make sense. :-)